### DISCUSSION TITLE
Fluxcd is not able to pick the changes

### DISCUSSION THREAD

**Discussion by Subhasish5** [fluxcd/flux2#3993] [Q&A] [answered]
Fluxcd is not able to pick the changes whenever we are making the commits to master branch.

![image](https://github.com/fluxcd/flux2/assets/46413003/b6ab4129-2b2e-4841-b081-ffadcc575b39)
The issue is : "client rate limiter Wait returned an error: context deadline exceeded"

Even after making a few changes to the github repository in the master branch, fluxcd is not able to pick the changes...

Also when we are trying to reconcile flux manually also, it's showing this error.

Please help.

**Comment by kingdonb** [comment, marked as answer]
There is not enough information to follow up this report, and this forum is not really intended for debugging one-off issues like this. Please consult the support guidelines for info about better more appropriate ways to seek out support for your deployment.

https://fluxcd.io/support/#getting-help-from-the-flux-community

Thanks for using Flux, and I hope you get to solve your issue!
